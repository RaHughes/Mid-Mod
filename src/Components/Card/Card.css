.card {
  margin: 2%;
  display: flex;
  flex-direction: column;
  height: 250px;
  width: 300px;
  background-color: white;
  border: 3px solid teal;
}

p {
  color: teal;
}

h1 {
  color: teal;
}

button {
  background-color: teal;
  color: white;
  height: 30px;
}