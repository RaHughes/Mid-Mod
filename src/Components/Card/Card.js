import React from 'react'
import './Card.css'

const Card = ({ name, date, time, number }) => {
return (
  <div className='card'>
    <h1>{name}</h1>
    <p>Date: {date}</p>
    <p>Time: {time}PM</p>
    <p>Number of Guests: {number}</p>
    <button>Cancel</button>
  </div>
)
}

export default Card;