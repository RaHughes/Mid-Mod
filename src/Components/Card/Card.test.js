import React from 'react';
import { shallow } from 'enzyme';
import Card from './Card'

describe('Card', () => {
  it('Should match snapshot', () => {
    const wrapper = shallow(<Card />);
    expect(wrapper).toMatchSnapshot();
  })
})