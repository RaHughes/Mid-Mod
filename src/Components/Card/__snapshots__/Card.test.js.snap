// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Should match snapshot 1`] = `
<div
  className="card"
>
  <h1 />
  <p>
    Date: 
  </p>
  <p>
    Time: 
    PM
  </p>
  <p>
    Number of Guests: 
  </p>
  <button>
    Cancel
  </button>
</div>
`;
